protocols/virtualdefund/virtualdefund.go

```diff
@@ -217,6 +248,71 @@ func (o Objective) fullySigned() bool {
 	return true
 }
 
+// isAlice returns true if the receiver represents participant 0 in the virtualdefund protocol
+func (o Objective) isAlice() bool {
+	return o.MyRole == 0
+}
+
+// isBob returns true if the receiver represents participant 2 in the virtualdefund protocol
+func (o Objective) isBob() bool {
+	return o.MyRole == 2
+}
```

Comment:
Occurs to me that there is scope for sharing production code between virtualfund and virtualdefund (as well as test code).